src/app/mainPage/rankingTable/RankingTable.tsx

```diff
@@ -0,0 +1,49 @@
+import { Typography } from '@mui/material';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+
+import { useLocale } from 'hooks';
+import { Container } from 'ui';
+import { styles } from './RankingTable.styles';
+import { RankingTableProps } from './RankingTable.types';
+
+export const RankigTable = ({ data }: RankingTableProps) => {
+  const { formatMessage } = useLocale();
+
+  return (
+    <Container>
+      <Typography>{formatMessage({ id: 'rankingTable.summary' })}</Typography>
+      <Table aria-label="simple table" sx={styles.table}>
+        <TableHead>
+          <TableRow>
+            <TableCell></TableCell>
+            <TableCell align="center">{formatMessage({ id: 'rankingTable.cycling' })}</TableCell>
+            <TableCell align="center">{formatMessage({ id: 'rankingTable.running' })}</TableCell>
+            <TableCell align="center">{formatMessage({ id: 'rankingTable.swimming' })}</TableCell>
+            <TableCell align="center">{formatMessage({ id: 'rankingTable.walking' })}</TableCell>
+            <TableCell align="center">{formatMessage({ id: 'rankingTable.others' })}</TableCell>
+            <TableCell align="center">{formatMessage({ id: 'rankingTable.pushUps' })}</TableCell>
```

Comment:
Hmm... a może lepiej zrobić jakiegoś enuma z tymi rodzajami i na jego podstawie z mapa to ładnie wszystko wygenerować? :)

Reply:
Jestem zażenowany sobą, że na to nie wpadłem